packages/tinacms/src/toolkit/fields/components/reference/reference-select.tsx

```diff
@@ -177,13 +177,7 @@ const ComboboxDemo: React.FC<ReferenceSelectProps> = ({
             shouldFilter={!field.experimental___filter}
             filter={(value, search) => {
               //Replace / in the file path with empty string to make it searchable
```

Comment:
**question**: Is this comment still relevant?

Reply:
Done - removed redundant comment